### PR TITLE
Timeline empty state improvements

### PR DESCRIPTION
**Pull request by carlinisaacson**
https://github.com/infinitered/reactotron/issues/1359

- Timeline empty state improvements
  - Adds a button for the quick start guide
  - Adds a random developer joke

Contributors: @carlinisaacson @trevor-coleman @darinwilson 

DD321274-A94C-42AF-BCAF-AB9A6996B126

### CODE REVIEW

File: lib/reactotron-core-ui/src/components/RandomJoke/RandomJoke.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  },
+]
+
+export function RandomJoke() {
+  const jokeRef = useRef(0)
+
+  useEffect(() => {
+    const ref = Math.floor(Math.random() * jokes.length)
+    jokeRef.current = ref
+  })
```
Comment (robinheinze) [inline lib/reactotron-core-ui/src/components/RandomJoke/RandomJoke.tsx:65]: Question: Is the dependency array left out here intentionally so a new joke will appear on every re-render?

**Review by robinheinze — APPROVED** [review]
Approved! Just left one question :)

File: lib/reactotron-core-ui/src/components/RandomJoke/RandomJoke.tsx (hunk 1 above)
Comment (carlinisaacson) [reply lib/reactotron-core-ui/src/components/RandomJoke/RandomJoke.tsx:65]: We wanted it to only run on mount. So we had an empty array. Then eslint was giving us a warning to either add the jokes.length dependency or remove the array. So we removed it. Then we extracted it to core-ui, and now when I add the empty array back I don't get the warning there. I personally would rather the empty array be there so I'll add it back in.

**Review by morganick — APPROVED** [review]
LGTM 👍 and works as expected.

Note: The screenshot on the docs needs to be updated: https://docs.infinite.red/reactotron/quick-start/react-native/

**Review by frankcalise — APPROVED** [review]
updated with latest from main

**Pull request merged**